Modix.Bot/Modules/ReplModule.cs

```diff
@@ -54,10 +54,7 @@ public class ReplModule : ModuleBase
         }
 
         [Command("exec"), Alias("eval", "e"), Summary("Executes the given C# code and returns the result.")]
-        public async Task ReplInvokeAsync(
-            [Remainder]
-            [Summary("The code to execute.")]
-                string code)
+        public async Task ReplInvokeAsync([Remainder, Summary("The code to execute.")] string code)
```

Comment:
How we format the attributes here is all personal preference of course, but changing it would make it inconsistent with the rest of the modules.

Reply:
Honestly this was just me getting annoyed by the auto-formatter that just wouldn't not leave the indentation alone, this was the end result 😛